spec/smoke/api/v1/providers_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper_smoke"
+
+describe "V1 Public API Smoke Tests", :aggregate_failures, smoke: true do
+  let(:recruitment_year) { Settings.current_recruitment_cycle_year }
+  let(:base_url) { Settings.publish_url.sub("www", "api") }
+
+  subject(:response) { HTTParty.get(url) }
+
+  let(:payload) { JSON(response.body).fetch("data", []) }
+
+  context "providers" do
+    describe "GET /v1/recruitment_cycles/:recruitment_year/providers" do
+      let(:url) { "#{base_url}/api/public/v1/recruitment_cycles/#{recruitment_year}/providers?[per_page]=1" }
+
+      it "returns a HTTP status code of 200" do
+        expect(response.code).to eq(200)
+      end
+
+      it "returns a payload with data" do
+        expect(payload.size).to be > 0
+      end
```

Comment:
@AbigailMcP Similar kinda question..


https://github.com/DFE-Digital/teacher-training-api/blob/f94a639c47afe1fd66c2495f0fd16d60c3fe3b25/spec/smoke/api/v1/courses_spec.rb#L18-L20

Reply:
Happy to go along with this. I did try `expect(payload).to have_at_least(1).provider` which is valid but needs another rspec matcher library.